src/alto-ui/Form/Radio/story.js

```diff
@@ -2,15 +2,22 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import centered from '@storybook/addon-centered';
+import { boolean } from '@storybook/addon-knobs';
 
 import Radio from './Radio';
 
 storiesOf('Form/Radio', module)
   .addDecorator(centered)
   .addWithJSX('overview', () => (
     <div>
-      <Radio id="inactive" label="Inactive" name="overview" />
-      <Radio id="checked" label="Checked" name="overview" checked />
```

Comment:
You should create two new radios. so on the storybook we can test the ones without readOnly.
on checked and other not

Reply:
Fixed